hal/src/main/native/include/HAL/handles/HandlesInternal.h

```diff
@@ -95,27 +142,67 @@ static inline int16_t getHandleTypedIndex(HAL_Handle handle,
  */
 
 // using a 16 bit value so we can store 0-255 and still report error
```

Comment:
Should this be below the public documentation or maybe inside the function?

Reply:
If its below its likely to mess up any documentation parser. So its probably good.